client/controllers/incidentReports/incidentForm.coffee

```diff
@@ -70,10 +74,14 @@ Template.incidentForm.onRendered ->
 
   @$('#add-incident').validator()
   #Update the validator when Blaze adds incident type related inputs
-  @autorun ->
-    instance.incidentType.get()
-    Meteor.defer ->
-      instance.$('#add-incident').validator('update')
+  @autorun =>
+    @incidentType.get()
+    Meteor.defer =>
+      @$('#add-incident').validator('update')
+
+  if not @incidentData.locations.length
```

Comment:
It would be better to prevent locations from being added to suggestedFields initially when there are no locations.